tests/test_licensemanager.py

```diff
@@ -0,0 +1,105 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from .common import BaseTest
+
+# from nose.tools import set_trace; set_trace()
+
+
+class TestLicenseConfiguration(BaseTest):
+
+    def test_tag_license_configuration(self):
+        session_factory = self.record_flight_data(
+            "test_license_configuration_tag"
+        )
+        p = self.load_policy(
+            {
+                "name": "tag-license-configuration",
+                "resource": "license-configuration",
+                "actions": [{"type": "tag", "key": "Category", "value": "TestValue"}],
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        for x in range(len(resources)):
+            self.assertEqual(len(resources), 1)
+
+    def test_remove_tag_license_configuration(self):
+        session_factory = self.record_flight_data(
+            "test_remove_tag_license_configuration"
+        )
+        p = self.load_policy(
+            {
+                "name": "untag-license-configuration",
+                "resource": "license-configuration",
+                "filters": [{"tag:Category": "TestValue"}],
+                "actions": [{"type": "remove-tag", "tags": ["Category"]}],
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+
+    def test_update_service_settings_license_configuration(self):
+        session_factory = self.record_flight_data(
+            "test_update_service_settings_license_configuration")
+        p = self.load_policy(
+            {
+                "name": "update-service-settings",
+                "resource": "service-settings",
+                # "Filters" : "None"
+            },
+            session_factory=session_factory,
+
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+
+    def test_update_license_configuration(self):
+        session_factory = self.record_flight_data("test_update_license_configuration")
+        p = self.load_policy(
+            {
+                "name": "update-license-configuration",
+                "resource": "license-configuration",
+                # "Filters" : "None"
+                "actions": [
+                    {
+                        "type": "update-license-configuration",
+                        "Name": "Database License",
+                        "Description": "testing",
+                        "LicenseCount": 500
+                    }
+                ],
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+        # client = session_factory().client("license-manager")
+
+    def test_delete_license(self):
```

Comment:
not a valid unit test, I could remove the action for delete and it would still pass.